src/components/Calculator.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Linters are exceptionally great tools that automatically detect and flag programmatic/stylistic errors. Instead of disabling these awesome tools, rather fix :wrench:  them.

Reply:
Thank you so much, i have worked on the changes.